log4j-core/src/main/java/org/apache/logging/log4j/core/Logger.java

```diff
@@ -53,6 +53,9 @@ public class Logger extends AbstractLogger implements Supplier<LoggerConfig> {
 
     private static final long serialVersionUID = 1L;
 
+    // Used to avoid allocations for empty varargs
+    private static final Object[] EMPTY_VARARGS = new Object[0];
```

Comment:
Use can can use the short cut {} on the right-hand side.

Reply:
Thanks for the quick review! updated